bundles/CoreBundle/EventListener/LegacyTemplateListener.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+namespace Pimcore\Bundle\CoreBundle\EventListener;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Event\GetResponseForControllerResultEvent;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Sensio\Bundle\FrameworkExtraBundle\Templating\TemplateGuesser;
+use Sensio\Bundle\FrameworkExtraBundle\EventListener\TemplateListener;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides backward compatibility for PHP templates
+ *
+ */
+class LegacyTemplateListener extends TemplateListener
+{
+    /**
+     * @var ContainerInterface
+     */
+    protected $container;
+
+    public function __construct(TemplateGuesser $templateGuesser, \Twig_Environment $twig = null, ContainerInterface $container)
+    {
+        $this->container = $container;

```

Comment:
Can also replaced by DelegatingEngine $templateEngine?

Reply:
I tried but tests were failing...need to debug!